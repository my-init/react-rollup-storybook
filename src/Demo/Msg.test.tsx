import React from 'react';
import { create, ReactTestRenderer } from 'react-test-renderer';

import Msg from './Msg';

let snapshot: ReactTestRenderer;

const MSG = 'Test MSG...';
const TAIL = 'lovely tail❤️';
beforeEach(() => {
  const msg = <Msg msg={MSG} tail={TAIL} />;

  snapshot = create(msg);
});

describe('<Msg />', () => {
  it('it matches the snapshot', () => {
    expect(snapshot.toJSON()).toMatchSnapshot();
  });
});


