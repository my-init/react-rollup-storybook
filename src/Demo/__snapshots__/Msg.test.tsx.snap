// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Msg /> it matches the snapshot 1`] = `
<div
  className="Msg"
>
  Test MSG...
  <p
    style={
      Object {
        "color": "#cfcfcf",
        "textAlign": "right",
      }
    }
  >
    <i>
      ———— lovely tail❤️
    </i>
  </p>
</div>
`;

exports[`<Msg /> it should have right props 1`] = `
Object {
  "children": Array [
    "Test MSG...",
    <p
      style={
        Object {
          "color": "#cfcfcf",
          "textAlign": "right",
        }
      }
    >
      <i>
        ———— lovely tail❤️
      </i>
    </p>,
  ],
  "className": "Msg",
}
`;
